.gitignore

```diff
@@ -1,25 +1,45 @@
-*.pyc
-# Generated by django compilemessages utility
+# This file enumerates various files/patterns to ignore in git.
+# Please keep this file tidy and neat
+
+# Local environment file
+.env
+
+# Django translation files
 *.mo
-wasa2il/locale/wasa2il.pot
+
+# Sqlite DB files
+*.sqlite
+
+# Log files
+*.log
```

Comment:
This is one thing I added that wasn't there before.
It seems common that log files are ignored across the board, since it seems unlikely that we'd ever want one in the repository.